packages/twenty-server/src/database/typeorm/core/migrations/1711557405330-addMissingMigration.ts

```diff
@@ -0,0 +1,102 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
```

Comment:
Other cascade condition to fix:

[twenty-server]  
   InternalServerErrorException: GraphQL errors on deleteFromworkspaceMember: {"message":"null value in column \"accountOwnerId\" of relation \"connectedAccount\" violates not-null constraint"}